src/cgEventVirtualKeyboard.mm

```diff
@@ -341,6 +377,33 @@ void sendCGEvent(int type, int code, int value)
             setModifierUp(code);
         }
     }
+    // Set the capslock modifier
+    if (code == KEY_CAPSLOCK)

```

Comment:
maybe we don't need to handle it in this PR, but it would nice to update the LED light of the caps lock button if `kCGEventFlagMaskAlphaShift` is in the modifiers. 

Looks like this function `manipulate_led` might do it

https://web.archive.org/web/20180924040224/http://osxbook.com/book/bonus/chapter10/kbdleds/download/keyboard_leds.c